tests/stub/server_side_routing/test_server_side_routing.py

```diff
@@ -70,14 +50,22 @@ def test_direct_connection_without_url_params(self):
     def test_direct_connection_with_url_params(self):
         params = "region=china&policy=my_policy"
         uri = "bolt://%s?%s" % (self._server.address, params)
-        self._server.start(script=direct_connection_without_routing_ssr_script,
-                           vars={
-                               "#VERSION#": "4.1"
-                           })
+        self._start_server()
         try:
             driver = Driver(self._backend, uri, self._auth, self._userAgent)
-        except Exception:
+        except types.DriverError:
             pass
+        except types.BackendError:
+            if get_driver_name() in ["javascript"]:
+                # TODO: this shouldn't be communicated as backend error
+                return
+        except Exception as e:
+            if get_driver_name() in ["java"]:
+                if "TestkitErrorResponse" in str(e):
```

Comment:
This is a general backend error. The underlying cause is:
```
java.lang.IllegalArgumentException: Routing parameters are not supported with scheme 'bolt'. Given URI: 'bolt://localhost:9001?region=china&policy=my_policy'
```

Reply:
Yeah, but that message type the Java back end sends, does not exist. Only `DriverError` and `BackendError` are defined in the testkit protocol (technically also `FrontendError` but that one is irrelevant here). In this case I'd expect a `DriverError` to be returned as it's the driver code that complains about what's going on here.

Reply:
I will note this in a card and should be able to fix it.